lib/adapters/class-adapter.php

```diff
@@ -399,6 +406,13 @@ public function enable_category_aggregation(): void {
 		$this->aggregate_categories = true;
 	}
 
+	/**
+	 * Enables faceting on empty search query strings.
+	 */
+	public function enable_empty_search_faceting(): void {
```

Comment:
since it is likely that we would want to have faceting _on_ in most cases, perhaps this should be changed to a disable function and the option defaults to true.

Reply:
to address your question more directly, I don't think a `disable` function is needed since this is something that would either be on or off for the full duration of a request lifecycle.

Reply:
The defaults are also different by platform. SearchPress, for example, has this on by default, whereas ElasticPress / VIP Enterprise Search has it off by default.